api/src/main/kotlin/nl/avisi/anowalke/TranslateConnector.kt

```diff
@@ -0,0 +1,45 @@
+package nl.avisi.anowalke
+
+import com.google.gson.Gson
+import mu.KLogger
+import mu.KotlinLogging
+import nl.avisi.anowalke.dto.ExpressionsDto
+import nl.avisi.anowalke.util.configureObjectMapper
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.ResponseEntity
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
+import org.springframework.web.client.RestTemplate
+
+interface TranslateConnector {
+    val api_key: String?
```

Comment:
Zijn die echt generieke variabelen of zijn deze vooral van toepassing voor de specifieke Microsoft Implementatie?

Reply:
Over het algemeen als je een externe api wil aanroepen heb je toch op z'n minst een key en een url nodig was ik van mening. Ze zijn well nullable dus je hoeft ze niet per se te gebruiken. Ik had de region er wel buiten gelaten want die is niet echt common.